changes/ocd-4554.sql

```diff
@@ -0,0 +1,51 @@
+-- Remove old 'statuses' field from Developer
+UPDATE openchpl.activity
+SET original_data = (original_data::jsonb - 'statuses')::text
+WHERE activity_object_concept_id = 3
+AND original_data IS NOT NULL;
+
+UPDATE openchpl.activity
+SET new_data = (new_data::jsonb - 'statuses')::text
+WHERE activity_object_concept_id = 3
+AND new_data IS NOT NULL;
+
+-- Remove the duplicate developer status accidentally created
+-- We are leaving the data from the "first click" and deleting from the "second click"
+DELETE FROM openchpl.vendor_status_history WHERE vendor_status_history_id = 2188;
+DELETE FROM openchpl.questionable_activity_developer WHERE id = 310;
+DELETE FROM openchpl.activity WHERE activity_id = 108600;
+
+---
+--- OCD-4554
+---
+DROP VIEW openchpl.certified_product_details;
+DROP VIEW openchpl.developer_search;
+
+-- add columns for start/end dates
+ALTER TABLE openchpl.vendor_status_history ADD COLUMN IF NOT EXISTS start_date date;
+ALTER TABLE openchpl.vendor_status_history ADD COLUMN IF NOT EXISTS end_date date;
```

Comment:
I thought our convention was to use `_day` for date values, and `_date` for timestamps, but I could be completely wrong

Reply:
It's.... inconsistent. The RWT fields are `_date`, ACB retirement date is `_date`, surveillance is `_date`, but certification criterion start/end is `_day` as well as functionality tested, etc. Are we moving to `_day`?